Packs/CommonPlaybooks/TestPlaybooks/playbook-Retrieve_File_from_Endpoint_-_Generic_V3_Test.yml

```diff
@@ -0,0 +1,307 @@
+id: Retrieve File from Endpoint - Generic V3 Test
```

Comment:
Did you intend for this to be a new test for v3? If so, please add also to `conf.json` with the relevant test playbook and integration needed.